src/components/Table/components/TableHead/index.tsx

```diff
@@ -57,7 +58,7 @@ export default function TableHead<D extends object>({ headerGroups, styles }: Ta
                             column.id !== '_all_' && (
                                 <TableCell {...column.getHeaderProps!()}>
                                     <Stack spacing="xs">
-                                        {column.canSort ? (
+                                        {column.canSort && column.id !== SELECTION_COLUMN_NAME ? (
```

Comment:
should we test for EXPAND_COLUMN_NAME as well?

Reply:
Actually we do not need this anymore, will remove it.